### PR TITLE
Fix newer boost build with c+11 AND Fix packaging

### PR DESCRIPTION
**Pull request by ShapeShifter499**
This is to fix the issue brought up at https://github.com/nochowderforyou/clams/issues/293

Also this is to fix some of the packaging that still refers to Novacoin

### CODE REVIEW

File: src/chainparams.cpp [hunk 1]
```diff
@@ -148,11 +148,11 @@ class CTestNetParams : public CMainParams {
         vFixedSeeds.clear();
         vSeeds.clear();
 
-        base58Prefixes[PUBKEY_ADDRESS] = list_of(111);
-        base58Prefixes[SCRIPT_ADDRESS] = list_of(196);
-        base58Prefixes[SECRET_KEY]     = list_of(239);
-        base58Prefixes[EXT_PUBLIC_KEY] = list_of(0x04)(0x35)(0x87)(0xCF);
-        base58Prefixes[EXT_SECRET_KEY] = list_of(0x04)(0x35)(0x83)(0x94);
+        base58Prefixes[PUBKEY_ADDRESS] = std::vector<unsigned char>(1,111);
+        base58Prefixes[SCRIPT_ADDRESS] = std::vector<unsigned char>(1,96);
```
Comment (Kefkius) [inline src/chainparams.cpp]: Should be 196, no?

**Comment by ShapeShifter499** [comment]
Fixed

File: src/chainparams.cpp (hunk 1 above)
Comment (ShapeShifter499) [inline src/chainparams.cpp]: I updated it

**Comment by l0rdicon** [comment]
awesome work. It looks good. I'll get this merged in tonight / tomorrow 

Thanks kef for picking up on the typo!

**Comment by ShapeShifter499** [comment]
Ok I'm fairly certain I've updated and fixed everything I needed to fix. Let me know if there are any other issues.

**Comment by ShapeShifter499** [comment]
Ok I'm fairly certain I've updated and fixed everything I needed to fix. Let me know if there are any other issues.

**Comment by kwentzell** [comment]
Tested ACK with clamd and clam-qt, will close this. These are good changes.

**Pull request merged**